certification/internal/shell/has_prohibited_packages.go

```diff
@@ -1,65 +1,28 @@
 package shell
 
 import (
-	"bufio"
-	"strings"
-
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
 	log "github.com/sirupsen/logrus"
 )
 
 type HasNoProhibitedPackagesCheck struct{}
 
 func (p *HasNoProhibitedPackagesCheck) Validate(image string) (bool, error) {
-	line, err := p.getDataToValidate(image)
+	result, err := podmanEngine.Unshare("HasNoProhibitedPackagesMounted", image, "./preflight", "check", "run")
```

Comment:
Hmm... can we assume this is where `preflight` is on a given system? Mine, for example, is `../preflight`. Some people might have it in path.

Reply:
Good point. I think I have a solution.